filesystem/unix_permission.go

```diff
@@ -11,8 +11,85 @@ import (
 	"syscall"
 )
 
+// whose permission should exist (for this file's permission) = condition (... file permission... Owner is root, Owner Can Read, Non Owner Cannot read
+// condition = string condition.? or OwnerIsRoot(uint) or OwnerOfFileIsRoot(filename).
+// whose permission should not exist
+// There are all rules.
+
+// somewhere else, get permission Mode
+// create rule: root is owner (condition. bitwise evaluator? or filename. filename!
+
+// OwnerRoot confition => call CheckFileOwnerRights
+// OwnerReadPermission => get Mode() => wrapper function like OwnerRead..
+//
+
 // CheckFileRights check that the given file path has been protected by the owner.
```

Comment:
This comment needs to be moved back above the CheckFileRights function. It's a godoc comment